presto-tests/src/main/java/com/facebook/presto/tests/TestingPrestoClient.java

```diff
@@ -193,6 +193,9 @@ else if (type instanceof VarcharType) {
         else if (VARBINARY.equals(type)) {
             return value;
         }
+        else if (type.toString().equals("ObjectId")) { // So there's no circular dependency (presto-mongodb -> presto-tests -> presto-mongodb)
```

Comment:
@martint Not really sure where to go from here. 

The easiest option is to drop the tests that support the changes, which is probably why ObjectId was never tested to begin with, but I feel like that lacks due diligence.

From what I can tell, the only way to get the tests working without making presto-tests aware of connector custom types, is to do a sizable refactor on presto-tests as a whole.
* `TestingPrestoClient` would need to be refactored to support being extended (static helpers would need to turn into protected class members
* `DistributedQueryRunner` would either:
 * need to some how optionally accept a client class reference during it's construction, so it can initialize the custom or default testing client or
 * need to be refactored so it may be extended without the entire class having to be duplicated to just replace the testing client.

Thoughts? 

I'm hoping not to let this fix get lost in the pile!

Thanks,
-Kevin